packages/plugins/auto-doc/src/components/data-table/data-table.svelte

```diff
@@ -0,0 +1,125 @@
+<script lang="ts">
+    import DataTable, { Head, Body, Row, Cell } from '@smui/data-table';
+    import Checkbox from '@smui/checkbox';
+    import  IconWrapper from '../../../../../ui/src/components/icons/icon-wrapper.svelte'
+    import IconButton from '@smui/icon-button'
+    import { type Template} from "@/pages/template-overview/template-overview.svelte"
+
+
+    export let allTemplates : Template[];
+
+    let selectedTemplates : Template[] = [];
+
+
+    function isInvalidDate(date: Date){
+        return isNaN(date.getTime());
+    }
```

Comment:
That's interesting, I don't know why it would attempt a type coercion since getTime() always returns a number in milliseconds.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getTime

Reply:
Yeah, as it seems it always return a number, in my opinion, I don't think it's necessary to append the Number.isNaN.
What do you think @michelguerin?